### ISSUE TITLE
postcss.config.cjs not modified correctly

### ISSUE THREAD

**Issue by zakkor** [svelte-add/tailwindcss#4]
Hi there, first of all thanks for the useful template!

On a fresh install of SvelteKit, the PostCSS adder, and then this Tailwind adder, the postcss.config.cjs file only gets partially replaced, the final version being this:
```
const tailwindcss = require("tailwindcss");
const autoprefixer = require("autoprefixer");
const cssnano = require("cssnano");

const mode = process.env.NODE_ENV;
const dev = mode === "development";

module.exports = {
	plugins: [
		autoprefixer,
		
		!dev && cssnano({
			preset: "default",
		}),
	],
};
```

https://github.com/svelte-add/tailwindcss/blob/main/preset.ts#L27-L35
It looks like it manages to add the `require` statement, but then the `.replace` call doesn't match for some reason and the `tailwindcss` plugin doesn't get added to the `plugins` array. I am running this on Windows if that's relevant.

**Comment by NoelJacob** [comment]
@zakkor Did you try checking if the line ending of postcss.config.js is CRLF instead of LF?

**Comment by zakkor** [comment]
I tthink it is indeed CRLF. Is the intended fix then to make the replacement regex work with both CRLF and LF?

**Comment by NoelJacob** [comment]
I usually keep 'core.autocrlf' to false in my git configuration and it solves the problem while running npx.



> I tthink it is indeed CRLF. Is the intended fix then to make the replacement regex work with both CRLF and LF?

Yup, I think so.

**Comment by babichjacob** [comment]
I forgot to mention—or test on my own—that I pushed an update to both presets that may or may not fix the problem?

**Comment by babichjacob** [comment]
Problem not actually fixed per #5 😓
